### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by raggleton** [raggleton/PomPom#2]
- [ ] Notification for the user: 
  - Sound?
  - Popup notification?
  - Flash screen/tab?
- [ ] Try and write in jQuery instead (just for the experience...)

**Comment by albyr** [comment]
Flashing the tab, or altering the favicon, is probably not noticeable enough. Sound is probably the best option.

**Comment by raggleton** [comment]
Yes, that probably sounds like the best method. Was also thinking about using notifications like Google calendar, as a mystery sound from a tab might be confusing/annoying.

**Comment by raggleton** [comment]
Just pushed a version with some sound effects when different state are activated. Bit basic, and so far only 3 sounds.

**Comment by albyr** [comment]
👍
